### ISSUE TITLE
UX Design Language for non-technical Users

### ISSUE THREAD

**Issue by xilsanctified** [EOSPortal/eosportal-front#47]
After validation from peers both locally and abroad, surfacing this here. Users on the technical end of the user funnel will have no problem reaching the point where they can vote leveraging the minimalistic front-end we have provided users. 

However, those who find technology more challenging will almost certainly run into trouble. While these concerns are loosely covered in the 'Help' section of the website -- what I am proposing is that we go above and beyond, leveraging a design language which focuses on the side of the user funnel whom lack technical skills.

As evidenced by the thousands of issues surfaced by users attempting to configure and register EOS addresses during the EOS token registration process -- voting has a high likelihood to suffer the same fate. 

As an action item, I would like to build out a mock-up (preferably in collaboration with others interested) of a user focused journey through the voting process, while laying a framework which will be simple for further UI development to take place as other pages / resources become necessary. Completely open to collaboration if the community decides this is worthy of the time.

Also happy to Merge Issue 35 and 36 into this effort.

**Comment by sogasg** [comment]
Maybe we could make it like a wizard, and have a button for the advanced mode for viewing the portal more like it stands today?

**Comment by xilsanctified** [comment]
Yeah, I like that idea. Given the web has shifted heavily towards a mobile first mentality, I would imagine the application would be styled in such a way that anyone, on any device, could easily browse the information outlined throughout the page. These formats tend to lend well to a 'spoon feed me information' mentality. So we could agree on what that looks like, and then offer an advanced mode which benefits from the same design language, but without the fluff around the edges.

Does anyone know if a mobile ready voting solution has been created? I'm not familiar with mobile + scatter solutions. @kesar Do you know?

**Comment by kesar** [comment]
> Does anyone know if a mobile ready voting solution has been created? I'm not familiar with mobile + scatter solutions. @kesar Do you know?

Its not possible unless yo do it in native. I know njames its working on Scatter mobile to use in apps, but I don't know when it will be ready, so I would focus in good experience for desktop users for now. 👍
